libpod/healthcheck.go

```diff
@@ -0,0 +1,62 @@
+package libpod
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"strings"
+)
+
+// HealthCheckStats represents the current state of a container
+type HealthCheckStatus int
+
+const (
+	HealthCheckSuccess           HealthCheckStatus = 0
+	HealthCheckFailure           HealthCheckStatus = 1
+	HealthCheckContainerStopped  HealthCheckStatus = 99
+	HealthCheckContainerNotFound HealthCheckStatus = 999
+	HealthCheckNotDefined        HealthCheckStatus = 9999
+)
+
+type HealthCheck struct {
```

Comment:
Why do we need this in addition to the manifest type we use in the container config?